README.md

```diff
@@ -244,24 +245,53 @@ Most router defaults should suffice, but you have full control over routing. See
 
 Options:
 * `router.pathDelimiter` (default: `"/:/"`) - Unique (uri-friendly) string to break apart image path, and image steps.
-* `router.cmdKeyDelimiter` (default: `"/"`) - Seperator between commands (aka image steps).
-* `router.cmdValDelimiter` (default: `"="`) - Seperator between a command and its parameters.
-* `router.paramKeyDelimiter` (default: `","`) - Seperator between command parameters.
-* `router.paramValDelimiter` (default: `":"`) - Seperator between a parameter key and its value.
+* `router.cmdKeyDelimiter` (default: `"/"`) - Separator between commands (aka image steps).
+* `router.cmdValDelimiter` (default: `"="`) - Separator between a command and its parameters.
+* `router.paramKeyDelimiter` (default: `","`) - Separator between command parameters.
+* `router.paramValDelimiter` (default: `":"`) - Separator between a parameter key and its value.
+* `router.signatureDelimiter` (default: `"/-/"`) - Separator between steps and the signed url
 * `router.originalSteps` (default: [Full Defaults](https://github.com/asilvas/node-image-steam/blob/master/lib/router/router-defaults.js)) - Steps performed on the original asset to optimize subsequent image processing operations. This can greatly improve the user experience for very large, uncompressed, or poorly compressed images.
 * `router.steps` (default: [Full Defaults](https://github.com/asilvas/node-image-steam/blob/master/lib/router/router-defaults.js)) - Mapping of URI image step commands and their parameters. This allows you to be as verbose or laconic as desired.
 
+## Security Options

```

Comment:
Great feature! But please move this section to bottom right before `# Things to try`